tests/Microsoft.Bot.Connector.Tests/MicrosoftGovernmentAppCredentialsTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Xunit;
+
+namespace Microsoft.Bot.Connector.Tests
+{
+    public class MicrosoftGovernmentAppCredentialsTests
+    {
+        [Fact]
+        public void MicrosoftGovernmentAppCredentials_Has_Gov_Endpoint()
```

Comment:
Given the amount of new code added, this doesn't seem like enough tests. 

Consider adding tests to cover a broader range of new behavior.

Reply:
done